tests/unit/FileUtilsTest.js

```diff
@@ -26,13 +27,13 @@ describe('FileUtils', () => {
         it('should append current time to the end of the file name', () => {
             const actualFileName = FileUtils.appendTimeToFileName('image.jpg');
             const expectedFileName = `image-${DateUtils.getDBTime()}.jpg`;
-            expect(actualFileName).toEqual(expectedFileName);
+            expect(actualFileName).toEqual(expectedFileName.replace(CONST.REGEX.ILLEGAL_FILENAME_CHARACTERS, '_'));
```

Comment:
I noticed that the test is using part of the same logic as the implementation, specifically the `.replace(CONST.REGEX.ILLEGAL_FILENAME_CHARACTERS, '_')`. This means it's essentially testing the implementation against itself, which can mask errors and lead to false positives.

A more robust approach would be to test against a static expectation, as shown below:

```js
        it('should append current time to the end of the file name', () => {
            const fixedTime = '2023-08-25 13:56:12.119';
            jest.spyOn(DateUtils, 'getDBTime').mockReturnValue(fixedTime);

            const actualFileName = FileUtils.appendTimeToFileName('image.jpg');
            const expectedFileName = 'image-2023-08-25 13_56_12.119.jpg'; // Static expectation

            expect(actualFileName).toEqual(expectedFileName);
        });
```

In this revised version, the expected file name is hardcoded, and a mock ensures that the time appended to the file name is known and fixed. This makes the test more robust and reliable.

By removing the `.replace(CONST.REGEX.ILLEGAL_FILENAME_CHARACTERS, '_')`, we ensure that if someone modifies the Regex in a way that introduces an error, the test has a chance to catch it. This change aligns the test more closely with the principle of testing the expected outcome rather than replicating the implementation logic.